### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by SimonBerend**
Please check structure and content of README.

### CODE REVIEW

**Comment by andrewsutjahjo** [comment]
Aaah, there's a little mixup here: you've added both the readme and the scraper code in both the readme and your own master branch.

This is a perfect opportunity to practice your git-fu: 
You've encapsulated your commits to be single units of work, which a good practice!
It also helps with untangling separate commits, which is what we'll need to do.

The eventual end goal is that your master branch only has the following commits:
```
adding scraper for startsomegood                             5c332ff
startsomegood scraper update.                                 1fbb8d0
processed comments on some good scraper.          50336e4
processed comments on some good scraper (2nd) 1ded13c
```
and your readme branch has the following commits:
```
added Intro, project aim, project set up to README.   7a62c08
more extensive project set up in README.                   0be39d8
```

This'll allow your reviewers to comment on only the scraper part, or only on the readme part. The golden rule is one feature, one PR. 
To put it in future perspective, stops horrible painful git conflicts where one feature that someone else has gotten onto master touches one of your features and you'll be in rebase hell for a significant amount of time (trust me, I've been there).

Just for good measure to make sure that you won't lose any work while you're learning, I'd like you to create a new branch from either readme or master called `safekeeping` and push that to your own github
```
git checkout -b safekeeping
git push
```

Then on both your readme and master branch do a 
```
git rebase -i HEAD~7
git push -f
```
The first command will give you an interface to either drop / pick / (lots of other options) commits. You can rewrite your own history based on which commits you want to keep(pick) and lose (drop). Then by force pushing, you can change your own branches to reflect that new history, and you'll have 2 PRs which only have the feature they say they have.

(Big note: this is good to do with your own branches, not good with master. )

**Comment by andrewsutjahjo** [comment]
btw, The readme is good. Usually readmes are light on organisation things (masterpeace / correlaid ) but I like it here. One comment is the docker stuff, isn't completely correct: docker is just an image that contains the operating system, code, and instructions to run each of the scrapers. The actual running will be scheduled using something on GCP, we haven't decided what yet really. But we'll have Marijke change that in the PR once she picks something :)

**Review by andrewsutjahjo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
-Forward 43 project between Correlaid and MasterPeace
+
+
+# Forward43 by MasterPeace in collaboration with CorrelAid
+
+**Forward43** is a data-driven project with the dream of supporting innovations worldwide by connecting them to decision-makers. We aim to create a match-making mechanism for decision-makers to better define their problems and connect them to �best-fit� innovations through using semantic match-making, machine learning modules, and purposeful Artificial Intelligence.
+
+The project aims to utilize vast data regarding innovations and find a smart framework to solve the world's most pressing issues facing societies, governments, organizations or businesses with current existing solutions.
+
+**MasterPeace** is a grassroots peace-building organization that believes that the world is �7 billion talents, not issues� Therefore, we, in MasterPeace, use the soft-power of music, art, sports and play to mobilize and inspire young people.  We believe that there is more that we have in common than what separates us.
```
Comment (MMAThijssen) [inline README.md:9]: Small comment: the dot is missing after "... billion talents, not issues".

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+
+We value open knowledge management and transparency in our work wherever possible while complying with GDPR regulations and following strong principles of data ethics.
+
+----------
+
+## Project aim
+
+Leading up to Forward43, MasterPeace found that their own database and the data available at other humanitarian, philatropic and entrepreneurial platforms offer relevant and in depth information on social innovations, which is often more valueble than the information found through conventional search engines (e.g. Google, Bing, Yahoo). Moreover, the teams involved became enthusiastic about gaining better insight into the landscape of social innovations. An overview of past and present projects enables social entrepreneurs to connect with like-minded people, build networks and learn from each other's experiences. Thus, in Forward43, we construct a search engine that enables MasterPeace chapters around the world to easily find and contact organisations and/or projects that have similar aims, operate in the same area, or have other relevant experience.
```
Comment (MMAThijssen) [inline README.md:24]: Typo: `valueble` should be `valuable`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+
+## Project set-up
+
+
+In order to build the search engine, we have created a pipeline to
+
+ 1.  Scrape social innovation data (Python)
+ 2. Automate periodic scraping (Docker)
+ 3. Collect the data in a database (Elasticsearch)
+ 4. Acces/search the database through dashboard/front end (Elasticsearch/...)
```
Comment (MMAThijssen) [inline README.md:37]: `Acces` should be `Access`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,74 @@
...
+
+
+#### 1. Scrape social innovation data (Python) 
+We collect data on social innovations and projects by scraping croudfunding and fundraising websites. Currently, we focus on the following sites:
```
Comment (MMAThijssen) [inline README.md:41]: `croudfunding` should be `crowdfunding`

**Pull request merged**